CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ if(POLICY CMP0051)
   cmake_policy(SET CMP0051 NEW)
 endif()
 
+if(POLICY CMP0054)
```

Comment:
LLVM has a minimum requirement for CMake at 3.4 IIRC.  The point is that it should already be NEW for everyone.  Why has no one hit this yet?  No one should be using 2.x.